src/vs/platform/userDataSync/common/userDataSyncStoreService.ts

```diff
@@ -458,6 +459,43 @@ export class UserDataSyncStoreClient extends Disposable {
 		this.clearSession();
 	}
 
+	async getLatestData(headers: IHeaders = {}): Promise<IUserDataSyncLatestData | null> {
+		if (!this.userDataSyncStoreUrl) {
+			throw new Error('No settings sync store url configured.');
+		}
+
+		const url = joinPath(this.userDataSyncStoreUrl, 'downloadLatest').toString();
```

Comment:
The API is at /v1/download/latest instead of /v1/download or /v1/downloadLatest.

Reply:
Fixing on the server side. Thanks!